internal/datastore/crdb/crdb.go

```diff
@@ -195,7 +199,8 @@ func (cds *crdbDatastore) Revision(ctx context.Context) (datastore.Revision, err
 	}
 
 	// Round the revision down to the nearest quantization
-	crdbNow := nowHLC.IntPart()
+	// Apply a delay to enable historial reads
+	crdbNow := nowHLC.IntPart() - cds.followerReadDelayNanos
 	quantized := crdbNow
 	if cds.quantizationNanos > 0 {
 		quantized -= (crdbNow % cds.quantizationNanos)
```

Comment:
Is there a risk that the quantization here pushes the window just slightly outside of the follower read and, if so, should we adjust the follower read delay accordingly?

Reply:
Any quantization will only add additional delay so the timestamp used for read will always be _at least_ the delay required. We could do some adjustment to reduce the overall amount of staleness by incorporating the quantization amount into the delay though but the difference should be minimal relative to the ~5s delay.